src/NUnitConsole/nunit3-console.tests/ResultReporterTests.cs

```diff
@@ -101,8 +101,9 @@ public void SummaryReportTest()
             var expected = new string[] {
                 "Test Run Summary",
                 "    Overall result: Failed",
-                "   Tests run: 35, Passed: 30, Errors: 1, Failures: 3, Inconclusive: 1",
-                "     Not run: 10, Invalid: 3, Ignored: 4, Explicit: 3, Skipped: 0",
+                "   Tests run: 35, Passed: 30, Failed: 7, Inconclusive: 1, Skipped: 7",
+                "     Failed Tests - Failures: 3, Errors: 1, Invalid: 3",
+                "     Skipped Tests - Ignored: 4, Explicit: 3, Other: 0",
                 "  Start time: 2015-10-19 02:12:28Z",
                 "    End time: 2015-10-19 02:12:29Z",
                 "    Duration: 0.349 seconds",
```

Comment:
@rprouse Sounds good to me. Sorry about the multiple commits, was struggling with some git issues earlier. I can squash them as soon as we have sorted all things out.
I will modify my changes according to your comment now.